lte/gateway/configs/templates/mme.conf.template

```diff
@@ -36,7 +36,7 @@ MME :
     USE_STATELESS = "{{ use_stateless }}";
     USE_HA = "{{ use_ha }}";
     ENABLE_GTPU_PRIVATE_IP_CORRECTION = "{{ enable_gtpu_private_ip_correction }}";
-    ENABLE_CONVERGED_CORE = "{{ enable_converged_core }}";
+    ENABLE_CONVERGED_CORE = "{{ enableConvergedCore }}";
```

Comment:
why camel case here?

Reply:
I am getting value in Camel Case format on AGW side. that's why Camel case here. Though I am cross checking it, Will update soon.

Reply:
I would be surprised if it's not camel case... no reason for it to be different from others in this file, though I could be wrong

Reply:
![image](https://user-images.githubusercontent.com/70897856/125912510-03b56a59-f626-4c3f-884f-debd4ab669c5.png)
Under proto file its in camel case. And I am getting updated value in that format only when I hit API on Orc8r side. A similar key(congestion_control_enabled) added on same api is not updating(/var/opt/magma/tmp/mme.conf file) for me. Let me Know if I am wrong.

Reply:
Did you validate that this works though (the templating)? I can't recall exactly, but I think we store mconfigs one way (camelCase) and consume it another (snake_case). Could be wrong, lmk

Reply:
Just noticed that other configs seem to be snake case, so was wondering. Maybe I didn't look closely enough though.

Reply:
I have rechecked this. I am getting updated value in camelCase only. I will check with AGW team and also with @ardzoht  on that, then will update here.